snowexsql/api.py

```diff
@@ -44,11 +46,14 @@ def get_points():
 
 class BaseDataset:
     MODEL = None
+    LINK_TABLE_MODEL = PointObservers
```

Comment:
unused constant?

The beauty of using SQLAlchemy's mapping is that we never have to worry about the link table, if we declared that with the two linked models. Think [this example](https://www.slingacademy.com/article/sqlalchemy-how-to-use-many-to-many-relationships/) is pretty well written (just look at the code)